heartbeat/monitors/active/http/_meta/fields.yml

```diff
@@ -8,15 +8,11 @@
         HTTP related fields.
       fields:
         - name: url
-          type: text
+          type: alias
+          path: url.full
+          migration: true
           description: >
             Service url used by monitor.
-          multi_fields:
```

Comment:
The type of the field changed here because of ECS. It means we don't have the indexed field anymore (`text`). Should this also be mentioned somewhere?